olive/model.py

```diff
@@ -290,6 +294,53 @@ def prepare_session(self, inference_settings: Dict[str, Any], device: Device, ra
 
         return get_ort_inference_session(self.model_path, inference_settings)
 
+    def nodes(self):
+        all_nodes = []
+        for graph in self.get_all_graphs():
+            for node in graph.node:
+                all_nodes.append(node)
+        return all_nodes
```

Comment:
yield better than list?
[https://stackoverflow.com/questions/7883962/where-to-use-yield-in-python-best](https://stackoverflow.com/questions/7883962/where-to-use-yield-in-python-best)
[https://www.techbeamers.com/when-prefer-yield-over-return-python/](https://www.techbeamers.com/when-prefer-yield-over-return-python/)

Reply:
Yeah since we only have iteration use case, I can make a change on this. This part of code is from onnxruntime https://github.com/microsoft/onnxruntime/blob/main/onnxruntime/python/tools/transformers/onnx_model.py#L72